Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -273,74 +273,73 @@ void EnhancementsWidget::SaveSettings()
 
 void EnhancementsWidget::AddDescriptions()
 {
-  static const char TR_INTERNAL_RESOLUTION_DESCRIPTION[] =
-      QT_TR_NOOP("Specifies the resolution used to render at. A high resolution greatly improves "
-                 "visual quality, but also greatly increases GPU load and can cause issues in "
-                 "certain games. Generally speaking, the lower the internal resolution is, the "
-                 "better your performance will be.\n\nIf unsure, select Native.");
-
-  static const char TR_ANTIALIAS_DESCRIPTION[] =
-      QT_TR_NOOP("Reduces the amount of aliasing caused by rasterizing 3D graphics. This smooths "
-                 "out jagged edges on objects.\nIncreases GPU load and sometimes causes graphical "
-                 "issues. SSAA is significantly more demanding than MSAA, but provides top quality "
-                 "geometry anti-aliasing and also applies anti-aliasing to lighting, shader "
-                 "effects, and textures.\n\nIf unsure, select None.");
+  static const char TR_INTERNAL_RESOLUTION_DESCRIPTION[] = QT_TR_NOOP(
+      "Specifies the resolution used to render at.\n\nA higher resolution greatly improves "
+      "visual quality, but also greatly increases GPU load and can cause issues in "
+      "certain games. Generally speaking, the lower the internal resolution is, the "
+      "better your performance will be.\n\nIf unsure, select Native.");
+
+  static const char TR_ANTIALIAS_DESCRIPTION[] = QT_TR_NOOP(
+      "Reduces the amount of aliasing caused by rasterizing 3D graphics. This smooths "
+      "out jagged edges on objects.\n\nIncreases GPU load and sometimes causes graphical "
+      "issues. SSAA is more demanding than MSAA, but provides top quality "
+      "geometry anti-aliasing and also applies anti-aliasing to lighting, shader "
```

Comment:
Put back "significantly", and drop the word "also" as "and also" is not really formal.

Reply:
I removed "significantly" as that word was all I needed to make your description on Linux not be too long. Hopefully removing the also's will balance out adding back "significantly"? I could remove the "anti-aliasing" from the "applies anti-aliasing to lighting, shader effects..." and it might fit again.

I went ahead and removed any also's from the other description texts, there was only one other.

Reply:
I checked and it fits fine the way I suggested.